torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -1163,15 +1163,14 @@ def test_batch_isend_irecv_tensor_err(self):
             self._barrier()
             rank = dist.get_rank()
             if rank == 0:
-                rank_to_GPU = self._init_multigpu_helper()
-                device_id = rank_to_GPU[rank][0]
                 with self.assertRaisesRegex(
                     RuntimeError, "Tensors must be CUDA and dense"
                 ):
                     send_tensor = _build_tensor(rank + 1)
                     send_op = dist.P2POp(dist.isend, send_tensor, 1)
-                    req = dist.batch_isend_irecv([send_op])
-                    req.wait()
+                    reqs = dist.batch_isend_irecv([send_op])
+                    for req in reqs:
+                        req.wait()
```

Comment:
I'm assuming this unit test passed since the code after `dist.batch_isend_irecv` was never executed and `batch_isend_irecv` just threw an exception. If so, should we just call `dist.batch_isend_irecv` here and ignore the return value?

Reply:
Yes, I agree, will update soon.